### ISSUE TITLE
“Create a Site” display is broken

### ISSUE THREAD

**Issue by zeldman** [Automattic/wp-calypso#33903]
In dialog intended to encourage new user to choose an appropriate theme for their business, if you choose BUSINESS, then WEB DESIGN, then select the “CALM” design option, the subhead text “Welcome! What can we do for you today?” is far too large for the containing block (especially in mobile preview view). The subhead is also set in white, and superimposed over a photo containing large swaths of white—useless and inaccessible choice. See screenshot:

![Screen Shot 2019-06-12 at 3 19 53 PM](https://user-images.githubusercontent.com/2983799/59390438-13161a00-8d26-11e9-8aa6-f079830bc36c.png)

#What should happen

• Text should fit inside the photo box
• White text against large white areas of photo are not appropriate; the design should be changed (or, more simply, a different background photo should be chosen).

#FIXTHEFLOWS

**Comment by jeffgolenski** [comment]
@fditrapani @michaeldcain @shaunandrews

**Comment by fditrapani** [comment]
Thanks for logging the issue @zeldman.  

> White text against large white areas of photo are not appropriate; the design should be changed (or, more simply, a different background photo should be chosen).

The white text, as it appears in your screenshot, was appearing under a transparent black overlay due to some new code that hadn't been merged yet. We shipped a fix that moves it to the top but even with that, I agree that it's still hard to read. We can make the overlay darker and pick a better image. This is what it looks like with the fix in place.

![image](https://user-images.githubusercontent.com/6981253/59465650-330a1400-8de0-11e9-984c-167f91bd412b.png)

> Text should fit inside the photo box

Looks like we have a regression here because we fixed this not too long ago.  cc @Automattic/zelda

**Comment by allancole** [comment]
This should probably be fixed directly in the theme. May make sense to move this issue to our themes repo: https://github.com/Automattic/themes/

**Comment by ramonjd** [comment]
> Looks like we have a regression here because we fixed this not too long ago

I wonder if it's a caching thing? We did updated the preview a few days ago

https://github.com/Automattic/wp-calypso/pull/33772
https://github.com/Automattic/wp-calypso/pull/33943

I checked proxied and unproxied:

<img width="1459" alt="Screen Shot 2019-06-14 at 9 02 23 am" src="https://user-images.githubusercontent.com/6458278/59472833-330f2000-8e83-11e9-8dc6-56e6d98c11f1.png">

Agree however that many of such things might be best addressed at the theme level. If we continue to add overrides in the onboarding flow, then it masks problems that the user may and probably will encounter after having created their site.

Let us know if this problem persists however, and we'll 👀 ASAP.

Cheers!

**Comment by millerf** [comment]
@ramonjd  I just tried and this issue seems to be resolved...

![Screenshot 2020-02-09 at 21 46 48](https://user-images.githubusercontent.com/790558/74109678-b48b8f00-4b85-11ea-9fbb-91fc11630912.png)
